drivers/base/bootvid/i386/pc98/bootvid.c

```diff
@@ -421,16 +423,17 @@ VidScreenToBufferBlt(
     ULONG X, Y;
     PUCHAR OutputBuffer;
     USHORT Px;
-    PUSHORT PixelsPosition = (PUSHORT)(FrameBuffer + FB_OFFSET(Left, Top));
+    PUSHORT PixelsPosition;
 
     /* Clear the destination buffer */
     RtlZeroMemory(Buffer, Delta * Height);
 
     for (Y = 0; Y < Height; Y++)
     {
         OutputBuffer = Buffer + Y * Delta;
+        PixelsPosition = (PUSHORT)(FrameBuffer + FB_OFFSET(Left, Top + Y));
 
-        for (X = 0; X < Width; X += 2)
+        for (X = 0; X < Width + 1; X += sizeof(USHORT))
```

Comment:
Why is the loop going from 0 up to "Width" included ? Shouldn't it be from 0 to Width-1 included (aka. to Width excluded) ?

Reply:
(Same comment reply as above)